macros/sql/get_column_values.sql

```diff
@@ -3,11 +3,9 @@
 {% endmacro %}
 
 {% macro default__get_column_values(table, column, order_by='count(*) desc', max_records=none, default=none) -%}
-{% if default is none %}
-    {% set default = [] %}
-{% endif %}
     {#-- Prevent querying of db in parsing mode. This works because this macro does not create any new refs. #}
     {%- if not execute -%}
+        {% set default = [] if not default %}
```

Comment:
return(default or []) is definitely less code, but I personally find this slightly more readable. Happy to change if lower code is important to you @joellabes 😄